src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -20,11 +41,7 @@ private unsafe void InitSort(CultureInfo culture)
             }
             else
             {
-                const uint LCMAP_SORTHANDLE = 0x20000000;
-
-                IntPtr handle;
-                int ret = Interop.Kernel32.LCMapStringEx(_sortName, LCMAP_SORTHANDLE, null, 0, &handle, IntPtr.Size, null, null, IntPtr.Zero);
-                _sortHandle = ret > 0 ? handle : IntPtr.Zero;
+                _sortHandle = GetSortHandle(_sortName);
```

Comment:
Nit: This can written as one-liner:

`_sortHandle = GlobalizationMode.Invariant ? GetSortHandle(_sortName) : IntPtr.Zero;`

Or even move the check for `GlobalizationMode.Invariant` into GetSortHandle method. It is duplicated in both callsites.

Reply:
I heard before when I was doing some stuff with the Range/Index that JIT can do better job when not using ```?``` operator. I am not sure if this was specific to that case or could be general. that is why I didn't use it here. I'll change it as you suggested.